packages/ide-extension/README.md

```diff
@@ -1,71 +1,29 @@
-    
- How to launch Guided Answer Extension from within another extension in VSCode or SBAS 
-     
-     1. Capture the Tree and Node Id from the Guide Answer's Guide URL that you wish to activate in the Guide Answers Extension
-     
-           https://ga.support.sap.com/dtp/viewer/index.html#/tree/**3046**/actions/**45995**
-
-     2. Activate Guide Answer's Extension by passing the required tree and node id where extension error occurs
-     
-     
- How to add a new guide to SAP's Guided Answers
+[SAP’s Guided Answers](https://ga.support.sap.com/dtp/viewer/index.html) is an interactive documentation designed to help troubleshoot issues, navigate processes, and guide users through tasks via a series of questions. SAP experts document the exact steps for analyzing issues and make that available to the customers. SAP's Guided Answers is recommended to customers to get the most from the support. Customers can look for solutions themselves instead of opening a customer support incident.
  
- 1. Navigate to  https://ga.support.sap.com/dtp/viewer/ and click on 'Contact Us' 
-`
-
-### How to Obtain Support  
-
-Guided Answers Extension
-
-To get more help, support, and information about the Guided Answers Extension please open a github issue 
-
-https://github.com/SAP/guided-answers-extension/issues/new?assignees=&labels=type%3Abug&template=bug_report.md&title=BUG+-+
-
-SAP's Guided Answers Support 
-
-To get more help, support and information about SAP's Guided Answer navigate to  https://ga.support.sap.com/dtp/viewer/ and click on 'Contact Us' 
-
-### How to Contribute 
-
-This project is open to feature requests/suggestions, bug reports etc. via [GitHub issues](https://github.com/SAP/guided-answers-extension/issues). Contribution and feedback are encouraged and always welcome. For more information about how to contribute, the project structure, as well as additional contribution information, see our [Contribution Guidelines](CONTRIBUTING.md).
-
+This Guided Answers extension by SAP renders content of already existing Guided Answers directly in an Integrated Developer Environment (IDE) such as SAP Business Application Studio (SBAS) and Microsoft Visual Studio Code (VSCode), making the content interactive and context-aware within the IDE. Developers do not need to navigate back and forth between separate supporting materials and their IDE when they come across an issue. This extension improves and speeds up the support process for SAP tools and increases user satisfaction. It will also help support teams provide clearer and more accurate solutions to issues to their customers. 
+## Usage 
+To launch Guided Answer Extension within VSCode or SBAS
+```
+1. Use Command Palette -> CMD/CTRL + Shift + P
+2. Type command -> SAP: Open Guided Answers
+```
+## Support
+
+To request a new guide, get help, support, or more information about SAP's Guided Answer navigate to  https://ga.support.sap.com/dtp/viewer/ and click on 'Contact Us'.
```

Comment:
This line is not needed in Guided Answers extension context

Reply:
removed with https://github.com/SAP/guided-answers-extension/pull/75/commits/f46731cb1db9404b2e6836ec1f4f5e98fe662184